### ISSUE TITLE
colorbox doesn't recognize gallery sets

### ISSUE THREAD

**Issue by slyhne** [Elgg/Elgg#8309]
Colorbox in Elgg 1.11.1 is unable to recognize gallery sets. So I'm missing prev/next on images in a set.

I have tried different things, like change colorbox opts to include:
slideshow: true and rel:  'market-galley'

Is it my markup that's wrong, or is it a bug?

HTML output:
```
    <div>
      <a href="http://www.example.com/market/image/147/1/master/1431517965.jpg" class="elgg-lightbox-photo market-thumbnail" rel="market-gallery">
        <img src="http://"http://www.example.com/market/image/147/1/small/1431517965.jpg" class="elgg-photo"/>
      </a>
      <a href="http://www.example.com/market/image/148/1/master/1431517975.jpg" class="elgg-lightbox-photo market-thumbnail" rel="market-gallery">
        <img src="http://"http://www.example.com/market/image/148/1/small/1431517975.jpg" class="elgg-photo"/>
      </a>
      <a href="http://www.example.com/market/image/149/1/master/1431517979.jpg" class="elgg-lightbox-photo market-thumbnail" rel="market-gallery">
        <img src="http://"http://www.example.com/market/image/149/1/small/1431517979.jpg" class="elgg-photo"/>
      </a>
    </div>
```

**Comment by mrclay** [comment]
Probably a bug in elgg.ui.lightbox. Does it work if you call `$('a.market-thumbnail').colorbox()` directly?

**Comment by slyhne** [comment]
That actually works, but first image comes up with this text besides the prev/next buttons:

Image {#index#} of {#count#}

After 2 seconds this text becomes:

Image 1 of 3

There is definitely something strange going on :)

**Comment by slyhne** [comment]
Maybe not so good a test I did. The first test was done with my modified js/lightbox/settings.php where I had added these two to colorbox opts:

slideshow: true,
rel: 'market-galley',

* With slideshow: true alone it doesn't work at all.
* With rel= 'market-galley' alone I got prev/next, but no option to stop/start slideshow.
* With both I get prev/next and start/stop slideshow.

The text I get first is the translated string, after 2 seconds these are replaced by what I believe is the built in text strings.

Strange...
